src/MacVim/MMWindowController.m

```diff
@@ -1665,5 +1684,10 @@ - (void)enterNativeFullScreen
     [decoratedWindow realToggleFullScreen:self];
 }
 
+- (void)processAfterWindowPresentedQueue
+{
+    [[NSOperationQueue mainQueue] addOperations:afterWindowPresentedQueue waitUntilFinished:NO];
```

Comment:
There's no point in using NSOperations or making a copy of newly created objects.